content/learn/book/welcome/_index.md

```diff
@@ -6,4 +6,44 @@ template = "book-section.html"
 page_template = "book-section.html"
 +++
 
-TODO: Briefly discuss what Bevy is.
+Welcome to the Bevy game engine! A bevy may be a group of birds, but our engine is:
+
+- **Rust-first:** with all the safety, tooling and power that comes with a modern systems language
+- **Completely free:** open source and hackable with an architecture designed for modularity
```

Comment:
This line makes me think of something like this SO https://softwareengineering.stackexchange.com/questions/21907/open-source-but-not-free-software-or-vice-versa

Reply:
Clarification added!